spread-shellcheck

```diff
@@ -181,14 +183,23 @@ def checkfile(path, executor):
     if errors:
         raise errors
 
+def is_file_in_dirs(file, dirs):
+    for dir in dirs:
+        if os.path.abspath(file).startswith('{}/'.format(os.path.abspath(dir))):
+            print('Skipping {}'.format(file))
```

Comment:
nitpick probably either the name of the function should change or the print message should be outside of this function since the name of the function suggests it is a generic function, but it has this print message in it which makes it less generic